_translator-files/po/fr/Installation-for-Macintosh.po

```diff
@@ -154,7 +153,7 @@ msgstr "Jetez un coup d'œil à la"
 #. type: Plain text
 #: ../wiki/en/Installation-for-Macintosh.md:42
 msgid "[Jamulus setup page](Getting-Started){: .button}"
```

Comment:
I wonder why the English isn't consistent here.  In several places, the page is called "Getting Started".  The file is "Getting Started".  Yet the text here is "Jamulus setup page".  @gilgongo any ideas?

Reply:
Probably that's an outdated reference to a page we deleted...

Reply:
Opened an issue.